.eslintrc.json

```diff
@@ -19,5 +19,8 @@
         "project": "./tsconfig.json"
       }
     }
+  },
+  "rules": {
+    "camelcase": "off"
```

Comment:
👀 may as well write the whole app in python now

Reply:
lmao, it pains me to do this. but we intentionally are not mapping the response for performance reasons.